extensions/wikia/ARecoveryEngine/ARecoveryEngineHooks.class.php

```diff
@@ -10,8 +10,14 @@ class ARecoveryEngineHooks {
 	 * @return bool
 	 */
 	public static function onWikiaSkinTopScripts( &$vars, &$scripts ) {
+		global $wgDevelEnvironment;
 
 		if ( ARecoveryModule::isEnabled() ) {
+			$spStatusURL = 'http://community.wikia.com/api/v1/ARecoveryEngine/SPStatus';
```

Comment:
Now, I wonder if we still have wikis which are not in `wikia.com` domain and how it'd work on those wikis.